extensions/android-paging/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1 @@
+<manifest package="com.squareup.sqldelight"/>
```

Comment:
this should have a different package like `com.squareup.sqldelight.android.paging`